db.php

```diff
@@ -1578,6 +1578,10 @@ public function ex_mysql_num_fields( $result ) {
 			return @mysql_num_fields( $result );
 		}
 
+		if ( ! $this->is_mysql_result( $result ) ) {
+			return false;
```

Comment:
Do we know what `return @mysqli_num_fields( $result );` would have previously returned on PHP 7.4 when the error was suppressed? If it's not `false`, we may want to use that instead.

And then also, do we want this conditional to happen first, above `if ( ! $this->use_mysqli )` in order to also protect against errors if mysqli isn't in use? Not sure if that case really even happens anymore though.

Reply:
For verbosity, this approach is no longer valid because we need to find out why `$this->result` is `null` in the first place in order to handle it correctly.